src/extension.ts

```diff
@@ -78,14 +81,29 @@ export async function activate(context: vscode.ExtensionContext): Promise<vscode
 	outputChannel.appendLine(`Registered command as part of sfdx-code-analyzer-vscode activation.`);
 	registerScanOnSave(outputChannel);
 	registerScanOnOpen(outputChannel);
+	await _stopExistingDfaRun(context, outputChannel, false);
```

Comment:
This feels out of place. Why are we calling to the "stop existing dfa run" method inside of the activate method?

Reply:
Synced up with @stephen-carter-at-sf offline and we decide to keep this simple for now. The cache would be at the workspace level and we let people run multiple graph engines from multiple vscode windows.